doc/release-notes/8880-Dvno-affiliation.md

```diff
@@ -0,0 +1,4 @@
+## New DB Settings
```

Comment:
Instead of the pull request number we usually use the issue number. So if you want to be a little more by the book, you can create an issue, use that number here, and mark the pull request to close that issue. All that said, this is also fine. 😄